bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/ResourceBundleTracker.java

```diff
@@ -30,16 +31,15 @@
  * @author Markus Rathgeb - Add locale provider support
  * @author Ana Dimova - fragments support
  */
-@SuppressWarnings("deprecation")
-public class ResourceBundleTracker extends ResolvedBundleTracker {
+@SuppressWarnings({ "deprecation", "rawtypes", "unchecked" })
```

Comment:
not for "rawtypes", as it complains about

```java
public class ResourceBundleTracker extends BundleTracker
```
so whatever you like better: living with the warning or with the too generous suppression, I'm fine with both